torch/csrc/jit/codegen/cuda/kernel_ir_dispatch.h

```diff
@@ -19,11 +19,6 @@ class IterDomain;
 class TensorDomain;
 class TensorView;
 
-namespace kir {
-
-// Vals
-class Predicate;
-class TensorIndex;
```

Comment:
The way github shows this change is a little misleading, it's moving:
```
// Exprs
class UnaryOp;
class BinaryOp;
class TernaryOp;
class ReductionOp;
class WelfordOp;
class BroadcastOp;
```
Out of the kir namespace.